bridge/engine/EngineMixer.cpp

```diff
@@ -283,7 +283,9 @@ void EngineMixer::addVideoStream(EngineVideoStream* engineVideoStream)
         engineVideoStream->_transport.getLoggableId().c_str(),
         endpointIdHash);
 
-    auto* outboundContext = obtainOutboundSsrcContext(*engineVideoStream, engineVideoStream->_localSsrc);
+    auto* outboundContext = obtainOutboundSsrcContext(*engineVideoStream,
+        engineVideoStream->_localSsrc,
+        engineVideoStream->_feedbackRtpMap);
     if (outboundContext)
     {
         engineVideoStream->_transport.getJobQueue().addJob<SetRtxProbeSourceJob>(engineVideoStream->_transport,
```

Comment:
It is actually in this setRtxProbe operation we should set also the rtpmap for rtx payload type. This should be set according to the SDP sent to the client.